### PR TITLE
Create or update baselines to match 2.0

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

File: src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json [hunk 1]
```diff
@@ -31,111 +31,6 @@
         }
       ],
       "GenericParameters": []
-    },
-    {
-      "Name": "Microsoft.Extensions.Logging.Configuration.ILoggerProviderConfiguration<T0>",
```
Comment (pranavkm) [inline src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json]: Isn't this a breaking change? Maybe @Eilon has more context?

File: src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "AssemblyIdentity": "Microsoft.Extensions.Logging.Configuration, Version=2.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```
Comment (ryanbrandenburg) [inline src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json]: The fact that this is listed as having been generated off of 2.1.0 seems concerning.

File: src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json (hunk 1 above)
Comment (Eilon) [reply src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json]: If the baseline appears to have a breaking change, does it even matter anymore? That would imply a break from 1.x to 2.0, so it's too late?

Comment (ryanbrandenburg) [reply src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json]: SO, `ILoggerProviderConfiguration` exists in the release/2.1 branch but does not exist in either the release/2.0 or release/1.1 branch. I'm pretty sure that as mentioned above, this baselines was accidentally generated against 2.1.0, and therefore most changes to it listed here don't mean much.

Comment (Eilon) [reply src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json]: Ah yes that presumably explains it.

File: src/Microsoft.Extensions.Logging.Testing/baseline.netcore.json [hunk 3]
```diff
@@ -1,5 +1,9 @@
 {
+<<<<<<< HEAD
```
Comment (Eilon) [inline src/Microsoft.Extensions.Logging.Testing/baseline.netcore.json]: Bad merge?

**Comment by ryanbrandenburg** [comment]
:up::date:

File: src/Microsoft.Extensions.Logging.Abstractions/breakingchanges.netcore.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "TypeId": "public class Microsoft.Extensions.Logging.Abstractions.NullLogger<T0> : Microsoft.Extensions.Logging.ILogger<T0>",
```
Comment (pranavkm) [inline src/Microsoft.Extensions.Logging.Abstractions/breakingchanges.netcore.json]: This type exists - https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging.Abstractions/NullLoggerOfT.cs. Why is it marked as removal?

Comment (pranavkm) [reply src/Microsoft.Extensions.Logging.Abstractions/breakingchanges.netcore.json]: Is this because the baseline was incorrect to start with?

Comment (Eilon) [reply src/Microsoft.Extensions.Logging.Abstractions/breakingchanges.netcore.json]: We think it's a bug in the tool, which is logged, and we have another issue to revert these "fake" breaking changes.

**Pull request merged**